lib/levelup.js

```diff
@@ -169,17 +161,13 @@ LevelUP.prototype.isClosed = function () {
 }
 
 LevelUP.prototype.get = function (key, options, callback) {
-  var promise
-
   callback = getCallback(options, callback)
+  callback = catering.fromCallback(callback)
 
-  if (!callback) {
-    callback = promisify()
-    promise = callback.promise
+  if (maybeError(this, callback)) {
+    return callback.promise
```

Comment:
Added a few extra line breaks here since the rest of the code seems to use that style.